.github/workflows/vib-thanos-bucketweb.yaml

```diff
@@ -0,0 +1,36 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    branches:
+      - master
+    paths:
+      - 'bitnami/thanos/*'
+env:
+  CSP_API_URL: https://console.cloud.vmware.com
+  CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+  VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
+jobs:
+  vib-k8s-verify: # verify in multiple target platforms
+    runs-on: ubuntu-latest
+    environment: vmware-image-builder
+    strategy:
+      matrix:
+       include:
+         - target-platform: gke
+           target-platform-id: 91d398a2-25c4-4cda-8732-75a3cfc179a1
+         - target-platform: aks
+           target-platform-id: 7b13a7bb-011c-474f-ad71-8152fc321b9e
+         - target-platform: tkg
+           target-platform-id: 7ddab896-2e4e-4d58-a501-f79897eba3a0
+      fail-fast: false
+    name: K8s verify ${{ matrix.target-platform}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+      - uses: vmware-labs/vmware-image-builder-action@main
+        with:
+          pipeline: thanos/bucketweb/vib-platform-verify.json
```

Comment:
I think there should only exist one pipeline definition, as there is no strong reason I can see for having them separated. This way, you would only need a single Kubernetes cluster provisioning 😁